ui/app/styles/components/tabs.scss

```diff
@@ -8,6 +8,17 @@
   ul {
     border-color: transparent;
     min-height: 3rem;
+
+    > a {
```

Comment:
the original styling for when it was inside a "li" element. Keeping that old styling just in case and duplicating it here when it's directly under a ul element.